scripts/xtensa-build-zephyr.py

```diff
@@ -931,10 +931,10 @@ def install_lib(sof_lib_dir, abs_build_dir, platform_wconfig):
 			dst = sof_lib_dir / llext_file
 
 			rimage_cfg = entry_path / 'rimage_config.toml'
-			llext_input = entry_path / (llext_base + '.so')
-			llext_output = entry_path / llext_file
+			llext_input = entry_path / (llext_base + '.llext')
+			llext_output = entry_path / (llext_file + '.ri')
```

Comment:
Err... no, please don't go from `.llext` to `.ri` and then BACK to `.llext` where each step is a different file; this is incredibly confusing!

One suggestion:

1. `.llext` because that's what comes from Zephyr so you don't have a choice?
2. `.ri0` because that's the first rimage stage. Similar to `zephyr_pre0`
3. Either `.ri` or `.llext.ri`.

Reply:
@marc-hb yeah, .llext -> something else -> .llext is ugly, I agree. There was a wish to have .llext as the final suffix, but now with the use of Zephyr cmake .llext comes out of those functions. Maybe yes - `.llext.ri` would be a good option for the final output